corehq/apps/userreports/tasks.py

```diff
@@ -194,6 +195,14 @@ def queue_async_indicators():
     cutoff = start + ASYNC_INDICATOR_QUEUE_TIME
     time_for_crit_section = ASYNC_INDICATOR_QUEUE_TIME.seconds - 10
 
+    yesterday = datetime.utcnow() - timedelta(days=1)
+    old_indicators = AsyncIndicator.objects.filter(date_queued__lt=yesterday).count()
+    if old_indicators:
+        notify_exception(
+            None,
+            message="{} indicators have been queued but did not complete in the past day".format(old_indicators)
+        )
```

Comment:
yeah that's way better gotta switch my thinking to using datadog first https://github.com/dimagi/commcare-hq/pull/15589/commits/4d928a55e586c752541c07c5f8b6547be44a9630